docs/source/markdown/podman-machine-init.1.md

```diff
@@ -35,6 +35,16 @@ Number of CPUs.
 
 Size of the disk for the guest VM in GB.
 
+#### **--extra-disk-num**=*number*, **-d**=*number*
+
+Number of extra disk(s) to add to the guest VM.
+Can only be increased. Only supported for QEMU machines.
```

Comment:
```suggestion
It can only be increased from the current setting. Only supported for QEMU machines.
```

Reply:
I'm fine with the suggestion, however, I simply used the same wording as the other sections. If I address this comment, the wording style will be inconsistent and I believe other sections will need to be changed. What do you think?

Reply:
If you could, I'd appreciate it being changed throughout this document.  If it's also on other pages, then I'd leave those.  But,  the original remark is just a suggestion, no foul if you decide not to take it.